djl-serving/multi-node/lws.yaml

```diff
@@ -82,17 +90,23 @@ spec:
             volumeMounts:
               - name: dshm
                 mountPath: /dev/shm
-              - name: model-storage
-                mountPath: /opt/ml/model
               - name: cache-storage
-                mountPath: /tmp/.djl.ai/python/
+                mountPath: /tmp/.djl.ai/
             env:
+              - name: DJL_LEADER_ADDR
+                value: "$(LWS_LEADER_ADDRESS)"
               - name: HF_MODEL_ID
```

Comment:
Do we need this on worker node?

Reply:
Yes, it's needed to report status to leader node.

Reply:
I mean do we need `HF_MODEL_ID` on worker node?

Reply:
Yes, it is used to download the model.